Lib/test/test_ssl.py

```diff
@@ -4251,7 +4258,7 @@ def test_pha_required_nocert(self):
         server_context.verify_mode = ssl.CERT_REQUIRED
         client_context.post_handshake_auth = True
 
-        server = ThreadedEchoServer(context=server_context, chatty=False)
+        server = ThreadedEchoServer(context=server_context, chatty=True)
```

Comment:
Remember to reset this default once all the tests are fixed.

Reply:
This is the other half of the 'peer did not return a certificate' error change above.
The 'tlsv13 alert certificate required' below was not being raised because of the error in server.
Looking at the other code in this test I don't know if I changed the test correctly but I think it illustrates what I'm seeing.

Reply:
I reworked this so that chatty=False.
I think the new commit makes more sense.